automatic/videoder/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,28 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$packageArgs = @{
+  packageName   = $env:ChocolateyPackageName
+  softwareName  = 'Videoder*'
+  fileType      = 'exe'
+  silentArgs    = '/S'
+  validExitCodes= @(@(0))
+}
+
+$uninstalled = $false
+
+[array]$key = Get-UninstallRegistryKey @packageArgs
+
+if ($key.Count -eq 1) {
+  $key | % {
+    $packageArgs['file'] = "$($_.UninstallString)"
```

Comment:
The return value from this registry item contains installer arguments as well, as such you will need to remove these arguments before it can be used as a file path.

The registry value will be something similar to `"C:\Program Files\Videoder\Uninstall Videoder.exe" /allusers`. (You can verify this yourself by downloading the snapshot archive and open up `.registry` in a text editor.)
https://ci.appveyor.com/project/chocolateycommunity/chocolatey-coreteampackages/builds/33144055/artifacts